ui/helpers/constants/zendesk-url.js

```diff
@@ -13,6 +13,29 @@ const ZENDESK_URLS = {
     'https://metamask.zendesk.com/hc/en-us/articles/4404722782107',
   CUSTOMIZE_NONCE:
     'https://metamask.zendesk.com/hc/en-us/articles/7417499333531-How-to-customize-a-transaction-nonce',
+  UNKNOWN_NETWORK:
+    'https://metamask.zendesk.com/hc/en-us/articles/4417500466971',
+  NFT_TOKENS:
+    'https://metamask.zendesk.com/hc/en-us/articles/360058238591-NFT-tokens-in-MetaMask-wallet',
+  USER_GUIDE_GAS:
+    'https://metamask.zendesk.com/hc/en-us/articles/4404600179227-User-Guide-Gas',
+  IMPORTED_ACCOUNTS:
+    'https://metamask.zendesk.com/hc/en-us/articles/360015289932',
+  USER_GUIDE_DAPPS:
+    'https://metamask.zendesk.com/hc/en-us/articles/4405506066331-User-guide-Dapps',
+  SPEEDUP_CANCEL:
+    'https://metamask.zendesk.com/hc/en-us/articles/360015489251-How-to-speed-up-or-cancel-a-pending-transaction',
+  VERIFY_CUSTOM_NETWORK:
+    'https://metamask.zendesk.com/hc/en-us/articles/360057142392',
+  USER_GUIDE_CUSTOM_NETWORKS:
+    'https://metamask.zendesk.com/hc/en-us/articles/4404424659995',
+  HARDWARE_CONNECTION:
+    'https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet',
+  BASIC_SAFETY:
+    'https://metamask.zendesk.com/hc/en-us/articles/360015489591-Basic-Safety-Tips',
+  INFURA_BLOCKAGE:
+    'https://metamask.zendesk.com/hc/en-us/articles/360059386712',
+  LEGACY_WEB3: 'https://metamask.zendesk.com/hc/en-us/articles/360053147012',
```

Comment:
Thank you for pulling these all together! 

Would be awesome to alphabetize these keys and add this file to our eslint sort rule. Can we alphabetize these here or open a follow-up PR for this? Let me know if you'd like me to open the follow-up PR proposed here as well

Reply:
Great suggestion! I'll sort the keys as part of this PR 👍

Reply:
Done!

Reply:
😄  tysm!